components/list/Item.tsx

```diff
@@ -54,30 +63,26 @@ export const Meta: React.FC<ListItemMetaProps> = ({
   );
 };
 
-export interface ListItemTypeProps extends React.FC<ListItemProps> {
+export interface ListItemTypeProps
+  extends ForwardRefExoticComponent<ListItemMetaProps & React.RefAttributes<HTMLElement>> {
```

Comment:
Hello! I'm having some issues after the release, I'm wondering if this should be `ListItemProps` instead of `ListItemMetaProps`?

Reply:
ref https://github.com/ant-design/ant-design/pull/35455